files/image_config/caclmgrd/caclmgrd

```diff
@@ -16,7 +16,9 @@ try:
     import subprocess
     import sys
     import syslog
-    from swsssdk import ConfigDBConnector
+    import sonic_device_util
```

Comment:
Add blank line between standard library imports and SONiC imports.

Reply:
@jleveque Done